components/certifications-rge-section/map-certicat-to-logo.tsx

```diff
@@ -0,0 +1,74 @@
+export const certificatLogo = {
```

Comment:
A lot of duplicates with `NomCertificat`and path here

Reply:
Yeah it's because some certifications share the same logo

Reply:
Or fo you think about an other thing, keep that point for a visio